app/components/publishers/vacancies_component.rb

```diff
@@ -30,14 +46,8 @@ def filters_applied_text
     I18n.t("jobs.dashboard_filters.heading", count: @filters[:managed_school_ids]&.count)
   end
 
-  def vacancy_type_tab_link(vacancy_type)
-    if vacancy_type == :awaiting_feedback
-      link_to jobs_with_type_organisation_path(vacancy_type), class: "govuk-tabs__tab" do
-        awaiting_feedback_tab(@organisation.vacancies.awaiting_feedback.count)
-      end
-    else
-      link_to t("jobs.#{vacancy_type}_jobs"), jobs_with_type_organisation_path(vacancy_type), class: "govuk-tabs__tab"
-    end
+  def vacancy_type_tab_link(vacancy_type, selected)
```

Comment:
Though I think you can't put a `?` in a parameter name

Reply:
Is there a reason to pass the `selected` as a parameter rather than directly calling the `selected` method within this method `vacancy_type_tab_link`?